packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -4,18 +4,21 @@
 
 import 'package:flutter/widgets.dart';
 
+import 'typography.dart';
+
 /// Interface for localized resource values for the material widgets.
 ///
 /// This class provides a default placeholder implementation that returns
 /// hard-coded American English values.
 class MaterialLocalizations {
   /// Create a placeholder object for the localized resources of material widgets
   /// which only provides American English strings.
-  const MaterialLocalizations();
+  const MaterialLocalizations({this.locale: const Locale('en', 'US')})
+      : assert(locale != null);
 
   /// The locale for which the values of this class's localized resources
   /// have been translated.
-  Locale get locale => const Locale('en', 'US');
+  final Locale locale;
```

Comment:
Why is this configurable? This class is the en-US localisation but it's also the interface that other MaterialLocalizations implementations will implement or extend, so it doesn't really make sense for this to be a field.

Reply:
Merged with Hans' code. No changes necessary for typography.